test/suite/test_debug_mode09.py

```diff
@@ -57,17 +57,3 @@ def test_update_restore_evict(self):
         pages_update_restored = stat_cursor[stat.conn.cache_write_restore][2]
```

Comment:
Looks good! Could you change the comment to saying "Read the statistics of pages that have been update restored"

Reply:
Sorry I didn't see this.